pyproject.toml

```diff
@@ -639,3 +648,51 @@ module = [
 ]
 disallow_untyped_defs = true
 # end: stronger typing
+
+# beginning: modules with possibly undefined variables
+[[tool.mypy.overrides]]
+module = [
+    "sentry.api.helpers.span_analysis",
```

Comment:
Does this mean we need to add new modules manually?
Is there a 'blocklist' approach instead so new stuff gets here automagically?

Reply:
this is the blocklist approach -- these modules are being ignored at the moment as they have possibly unbound variable issues.